src/mwoffliner.lib.ts

```diff
@@ -197,7 +197,7 @@ async function execute(argv: any) {
   }
   const mainPage = customMainPage || (articleList ? '' : mwMetaData.mainPage);
 
-  await downloader.checkCapabilities(mainPage);
+  await downloader.checkCapabilities(mainPage, mwMetaData.mainPage);
```

Comment:
Why adding a new argument? I thought we agre we should always test with `mwMetaData.mainPage`?

Reply:
that info is not available in downloader class need to pass that here

Reply:
ok I get it sorry